crystal_diffusion/models/model_loader.py

```diff
@@ -23,9 +24,17 @@ def load_diffusion_model(hyper_params: Dict[AnyStr, Any]) -> PositionDiffusionLi
     Returns:
         Diffusion model randomly initialized
     """
-    score_network_parameters = MLPScoreNetworkParameters(
+    score_network_architecture = hyper_params['model']['score_network'].get('architecture', 'mlp')
+    if score_network_architecture == 'mlp':
+        score_network_parameters_class = MLPScoreNetworkParameters
+    elif score_network_architecture == 'mace':
+        score_network_parameters_class = MACEScoreNetworkParameters
+    else:
+        raise NotImplementedError(f'Architecture {score_network_architecture} is not available.')
+
+    score_network_parameters = score_network_parameters_class(
         number_of_atoms=hyper_params['data']['max_atom'],
-        **hyper_params['model']['score_network']
+        **{k: v for k, v in hyper_params['model']['score_network'].items() if k != 'architecture'}
```

Comment:
This is fine for now.

I'd like to start introducing `SimpleParsing` from Fabrice@IDT (https://github.com/lebrice/SimpleParsing). I think we can do that bit by bit, as QoL bonuses on top of other work.